support-frontend/app/controllers/DigitalSubscriptionController.scala

```diff
@@ -86,13 +89,20 @@ class DigitalSubscriptionController(
             v2recaptchaConfigPublicKey,
             orderIsAGift,
             noindex = true,
-          ),
+          ) {
+            Html(
+              s"""<script type="text/javascript">
+                    window.guardian.productPrices = ${outputJson(productPrices(queryPromos, orderIsAGift))}
```

Comment:
This value seems to be [set in the `subscriptionsCheckout`](https://github.com/guardian/support-frontend/blob/main/support-frontend/app/views/subscriptionCheckout.scala.html#L40). Should we just be adding the `orderAsGift` to that method?

If this is the case, it should be a matter of just setting one more prop to the `subscriptionCheckout` for `promotionCopy`.